tests/customer-resource-show-test.js

```diff
@@ -77,9 +77,22 @@ describeApplication('CustomerResourceShow', function() {
 
     describe("selecting a package title to add to my holdings", function() {
       beforeEach(function() {
+        /*
+         * The expectations in the convergent `it` blocks
+         * get run once every 10ms.  We were seeing test flakiness
+         * when a toggle action dispatched and resolved before an
+         * expectation had the chance to run.  We sidestep this by
+         * temporarily increasing the mirage server's response time
+         * to 50ms.
```

Comment:
Let's put a `TODO` in here to explicitly control server timing directly from Mirage.